app/controllers/services_controller.rb

```diff
@@ -0,0 +1,44 @@
+###
+# A controller to handle any external services that need to be configured by an admin
+class ServicesController < Spotlight::ApplicationController
+  before_action :authenticate_user!
+  load_and_authorize_resource :exhibit, class: Spotlight::Exhibit
+  before_action :build_resource
+  load_and_authorize_resource :bibliography_service, through: :exhibit, singleton: true
+
+  def edit
+    add_breadcrumb t(:'spotlight.exhibits.breadcrumb', title: @exhibit.title), [spotlight, @exhibit]
+    add_breadcrumb t(:'spotlight.configuration.sidebar.header'), spotlight.exhibit_dashboard_path(@exhibit)
+    add_breadcrumb t(:'services.menu_link'), edit_exhibit_services_path(@exhibit)
+  end
+
+  def update
+    if @bibliography_service.update(update_params)
+      redirect_to edit_exhibit_services_path(@exhibit), notice: I18n.t('services.update.notice')
+    else
+      redirect_to edit_exhibit_services_path(@exhibit), alert: I18n.t('services.update.error')
+    end
+  end
+
+  def create
+    if @bibliography_service.update(update_params)
+      redirect_to edit_exhibit_services_path(@exhibit), notice: I18n.t('services.create.notice')
+    else
+      redirect_to edit_exhibit_services_path(@exhibit), alert: I18n.t('services.create.error')
+    end
+  end
+
+  private
+
+  def build_resource
+    @bibliography_service = if @exhibit.bibliography_service
+                              @exhibit.bibliography_service
+                            else
+                              ::BibliographyService.new(exhibit_id: @exhibit.id)
```

Comment:
Upon further reflection, I felt `find_or_initialize_by` was the behavior we wanted here, and implemented here: e9ec64d